steps/kobiton-app-upload/1.0.0/step.yml

```diff
@@ -0,0 +1,91 @@
+title: Kobiton App Upload
+summary: |
+  Uploads an application to Kobiton App Repo
+description: |
+  This step will take an .apk or .ipa file and upload it to the Kobiton App Repository. You can find your API key at [Kobiton](https://portal.kobiton.com/settings/keys)
+website: https://kobiton.com
+source_code_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo
+support_url: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo/issues
+published_at: 2021-12-02T21:29:30.928866+07:00
+source:
+  git: https://github.com/kobiton/bitrise-step-kobiton-upload-apps-repo.git
+  commit: 6ea2b2ee115463bee6398e693210af081628d012
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- kobiton_app_name: $BITRISE_APP_TITLE
+  opts:
+    is_expand: true
+    is_required: true
+    title: App Name
+- kobiton_app_path: $BITRISE_SIGNED_APK_PATH
+  opts:
+    is_expand: true
+    is_required: true
+    summary: Path to the app .apk or .ipa file
+    title: App Path
+- kobiton_app_id: null
```

Comment:
`: null` default values can be omitted

Reply:
I have omitted it in my repo. But don't know why when I use `bitrise share` to submit PR, It's changed from empty to `null`

Reply:
will remove it, anw

Reply:
We'll improve the `bitrise share` command, thanks!